### PR TITLE
EPCT-6 Wizard Component

### PR DESCRIPTION
**Pull request by edrisse**

### CODE REVIEW

File: app/common/patient/components/wizard/patientWizard.html [hunk 1]
```diff
@@ -4,59 +4,61 @@
...
+  <form name="vm.formNames" class="form-horizontal" novalidate>
+    <poc-wizard-step form="vm.formNames" title="{{'PATIENT_INFO_NAME' | translate}}">
+      <patient-names-step patient="vm.patient" form="vm.formNames" show-messages="vm.showMessages" />
+    </poc-wizard-step>
+  </form>
 
-  <div class="pull-right">
+  <form name="vm.formGender" class="form-horizontal" novalidate>
+    <poc-wizard-step form="vm.formGender" title="{{'PATIENT_INFO_GENDER' | translate}}">
+      <patient-gender-step patient="vm.patient" form="vm.formNames" show-messages="vm.showMessages" />
```
Comment (ynurmahomed) [inline app/common/patient/components/wizard/patientWizard.html]: It should be `form="vm.formGender"` right?

File: app/common/patient/components/wizard/steps/patientAddressStep.js [hunk 2]
```diff
@@ -5,7 +5,9 @@
     .module('common.patient')
     .component('patientAddressStep', {
       bindings: {
-        patient: '<'
+        patient: '<',
+        form: '<',
+        showMessages: '<'
       },
       controller: PatientAddressStepController,
       controllerAs: 'vm',
```
Comment (ynurmahomed) [inline app/common/patient/components/wizard/steps/patientAddressStep.js]: Looks like `require: {patientWizard: '^^',}` bellow is no longer required.

File: test/unitTests/spec/common/ui-helper/components/poc-wizard/pocWizard.spec.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    $urlRouterProvider.deferIntercept();
+  }));
+
+  beforeEach(inject((_$componentController_) => {
+    $componentController = _$componentController_;
+  }));
+
+  describe('addStep', () => {
+
+    it('infact adds the step', () => {
```
Comment (ynurmahomed) [inline test/unitTests/spec/common/ui-helper/components/poc-wizard/pocWizard.spec.js]: `it('should ...')`?

**Comment by edrisse** [comment]
done @ynurmahomed please review again

File: app/common/patient/components/wizard/steps/patientAddressStep.js (hunk 2 above)
Comment (ynurmahomed) [reply app/common/patient/components/wizard/steps/patientAddressStep.js]: @edrisse All components in `app/common/patient/wizard/steps` seem to be have `require: {patientWizard: '^^',}`.

Comment (edrisse) [reply app/common/patient/components/wizard/steps/patientAddressStep.js]: Done @ynurmahomed

**Pull request merged**